libraries/AP_EFI/AP_EFI.cpp

```diff
@@ -52,6 +53,14 @@ const AP_Param::GroupInfo AP_EFI::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_COEF2", 3, AP_EFI, coef2, 0),
 
+    // @Param: _FUEL_DENS
+    // @DisplayName: ECU Fuel Density
+    // @Description: Used to calculate fuel consumption for Currawong protocol.
+    // @Units: kg/m/m/m
+    // @User: Advanced
+    // @range: 0 3000
```

Comment:
Your current autotest failure is from here I think. 

You may want to try these tests locally first so you dont have to wait for CI to get to it. https://ardupilot.org/dev/docs/the-ardupilot-autotest-framework.html



```suggestion
    // @Range: 0 3000
```